### ISSUE TITLE
Forced fits strategy for non detections backwards and forwards in time

### ISSUE THREAD

**Issue by ajstewart** [askap-vast/vast-pipeline#17]
When a source either "disappears" or a new source is detected it is often vital to get information at the location of the source in images where it is not detected at that location.

Both VAST and TraP do this. VAST uses Aegean 'priorized` fitting (I think both backwards and forwards in time). It is an optional feature. TraP does force fits using PySE forward in time only after a new source is detected. It is optional for how many epochs to continue force-fitting the detected source if not detected.

TraP has a separate 'monitor' feature where you can ask it to do forced fits at a location throughout the entire run to build a light curve. Only downside of this feature is that you have to run the TraP again. 

The reason the options above are optional or have some optional element is that they are computationally expensive, especially when a typical TraP run of deep ASKAP images can produce thousands of new sources, most of which are rubbish. I think we can do better then the current 'new source analysis' that TraP does which may help this issue, but that's a separate issue in itself.

Another fast option could perhaps be to fetch Xsigma upper limits using the RMS maps (which are produced by default by ASKAPsoft).
